Libs/MRML/Core/vtkMRMLStorageNode.h

```diff
@@ -117,14 +117,17 @@ class VTK_MRML_EXPORT vtkMRMLStorageNode : public vtkMRMLNode
   /// Transferring: data is remote, and the transfer is working to completion
   /// TransferDone: the data is on disk and ready to be read
   /// Cancelled: the user cancelled the remote data transfer
+  /// CancelledNoData: reading was cancelled due to data not present. Can be
```

Comment:
This is for reading and writing, too, so talk about writing as well (see comment above). I would also make it more clear that it is not just "can be valid" but always valid and is used if and only if the data node is empty and there is nothing to be written or read.